src/routes/popup/swap/utils/swap.hooks.ts

```diff
@@ -375,9 +411,9 @@ export function usePoolQuote({
     } finally {
       setIsLoading(false);
     }
-  }, [tokenIn, tokenOut, slippage, amountIn, poolId, poolType, wanderFeePercent]);
+  }, [tokenIn, tokenOut, slippage, amountIn, poolId, poolType, wanderFee, stopFetching]);
 
-  useAsyncEffect(async () => {
+  useEffect(() => {
```

Comment:
`useAsyncEffect` doesn't clean the interval on unmount so used the `useEffect` but we can fix the `useAsyncEffect` hook as well later for cleanup.

Reply:
Let's add a `TODO:` on the `useAsyncEffect` so it's documented

Reply:
@7i7o https://github.com/wanderwallet/Wander/pull/1018